What will be the result of attempting to run the following program?

It will throw NullPointerBoundsException.
//There is no such exception.

It will throw ArrayIndexOutOfBoundsException.

It will print null.//true

It will run without any error but will print nothing.

None of the above.

Explanation
arr[0][1][2] => [0] = { { "a", "b" , "c"}, { "d", "e", null } },
[1] = { "d", "e", null } and [2] = null.
So it will print null.


